cscope build errors on Centos 7

cscope fails to build on Centos 7. The configure script output says it can't find curses. As a shot-in-the-dark I tried adding ```depends_on('ncurses')``` and get the same result

``` bash
$ spack install -j1 cscope
==> Installing cscope
==> ncurses is already installed in spack/opt/spack/linux-centos7-x86_64/gcc-4.8.5/ncurses-6.0-75hnzrecoib5eqdvfchl6n2jgcxc6dgo
==> Using cached archive: spack/var/spack/cache/cscope/cscope-15.8b.tar.gz
==> Staging archive: spack/var/spack/stage/cscope-15.8b-prz66vexnihyvq54as3ry3f6fucox2rk/cscope-15.8b.tar.gz
==> Created stage in spack/var/spack/stage/cscope-15.8b-prz66vexnihyvq54as3ry3f6fucox2rk
==> Ran patch() for cscope
==> Building cscope [AutotoolsPackage]
==> Executing phase : 'autoreconf'
==> Executing phase : 'configure'
==> Executing phase : 'build'
==> Error: ProcessError: Command exited with status 2:
    'make'
spack/lib/spack/spack/build_systems/autotools.py:273, in build:
     268      def build(self, spec, prefix):
     269          """Makes the build targets specified by
     270          :py:attr:``~.AutotoolsPackage.build_targets``
     271          """
     272          with working_dir(self.build_directory):
  >> 273              inspect.getmodule(self).make(*self.build_targets)

See build log for details:
  /tmp/spack-stage/spack-stage-RqIblB/cscope-15.8b/spack-build.out
```

From spack-build.out
``` bash
spack/lib/spack/env/gcc/gcc  -g -O2   -o cscope fscanner.o egrep.o alloc.o basename.o build.o command.o compath.o crossref.o dir.o display.o edit.o exec.o find.o help.o history.o input.o invlib.o logdir.o lookup.o main.o mouse.o mygetenv.o mypopen.o vpaccess.o vpfopen.o vpinit.o vpopen.o -lcurses
/bin/ld: cannot find -lcurses
collect2: error: ld returned 1 exit status
make[2]: *** [cscope] Error 1
make[2]: Leaving directory `/tmp/spack-stage/spack-stage-RqIblB/cscope-15.8b/src'
make[1]: *** [all-recursive] Error 1
make[1]: Leaving directory `/tmp/spack-stage/spack-stage-RqIblB/cscope-15.8b'
make: *** [all] Error 2
```

--- a/var/spack/repos/builtin/packages/cscope/package.py
+++ b/var/spack/repos/builtin/packages/cscope/package.py
@@ -39,9 +39,4 @@ class Cscope(AutotoolsPackage):
     depends_on('bison', type='build')
     depends_on('pkg-config', type='build')
 
-    def configure_args(self):
-        return [
-            'LIBS=-lncursesw',
-            '--with-ncurses={0}'.format(self.spec['ncurses'].prefix),
-            '--with-flex',
-        ]
+    build_targets = ['CURSES_LIBS=-lncursesw']

--- a/var/spack/repos/builtin/packages/cscope/package.py
+++ b/var/spack/repos/builtin/packages/cscope/package.py
@@ -33,12 +33,15 @@ class Cscope(AutotoolsPackage):
 
     version('15.8b', '8f9409a238ee313a96f9f87fe0f3b176')
 
-    variant('flex', default=False, description="Builds with flex")
+    depends_on('ncurses')
 
-    depends_on('flex', when='+flex')
+    depends_on('flex', type='build')
+    depends_on('bison', type='build')
+    depends_on('pkg-config', type='build')
 
     def configure_args(self):
-        args = []
-        if '+flex' in self.spec:
-            args.append('--with-flex')
-        return args
+        return [
+            'LIBS=-lncursesw',
+            '--with-ncurses={0}'.format(self.spec['ncurses'].prefix),
+            '--with-flex',
+        ]

